### DISCUSSION TITLE
Suggested function: Pitch trim reset via ground mode function after touchdown

### DISCUSSION THREAD

**Discussion by metronorth** [flybywiresim/aircraft#3090] [Ideas]
Assuming [this overview of Airbus flight control laws](http://www.airbusdriver.net/airbus_fltlaws.htm) is accurate and fully or mostly applicable to the A320-200 Neo, I figured I'd start a thread suggesting the pitch trim be reset after touchdown automatically. This might be tricky or even impossible depending on how much access the great devs of this project really have to the aircraft and it's systems. Thanks.

Snippet of what's contained in URL:

GroundMode  

- Active when aircraft is on the ground.
- Direct proportional relationship between the sidestick deflection and deflection of the flight controls.
- Is active until shortly after liftoff.After touchdown, ground mode is reactivated and resets the stabilizer trim to zero.

**Comment by ExampleWasTaken** [comment]
All things you mentioned are already implemented.
